lib/client/api_confluence.es6.js

```diff
@@ -0,0 +1,194 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+require('../confluence/api_velocity_data.es6.js');
+require('../confluence/failure_to_ship_data.es6.js');
+require('../confluence/vendor_specific_data.es6.js');
+require('../confluence/aggressive_removal_data.es6.js');
+
+foam.CLASS({
+  name: 'ApiConfluence',
+  package: 'org.chromium.apis.web',
+  documentation: `ApiConfluence is a client side object that
+    has methods to retrieve confluence metrics from Rest DAO such
+    as API velocity, failure-to-ship, aggressive removal and
+    vendor-specific-APIs.`,
+  requires: [
+    'foam.dao.ArraySink',
+    'foam.mlang.ExpressionsSingleton',
+    'org.chromium.apis.web.ApiVelocityData',
+    'org.chromium.apis.web.FailureToShipData',
+    'org.chromium.apis.web.VendorSpecificData',
+    'org.chromium.apis.web.AggressiveRemovalData',
+  ],
+  properties: [
+    {
+      name: 'apiVelocityDAO',
+      documentation: `A DAO that contains ApiVelocity objects.`,
+      typeName: 'ApiVelocity DAO',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'failureToShipDAO',
+      documentation: `A DAO that contains number of failur-to-ship APIs for
+        each browser associated with date.`,
+      typeName: 'failureToShipDataDAO',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'vendorSpecificDAO',
+      documentation: `A DAO that contains number of vendor-specific APIs for
+        each browser associated with date.`,
+      typeName: 'vendorSpecificDataDAO',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'aggressiveRemovalDAO',
+      documentation: `A DAO that contains number of aggressive removal APIs for
+        each browser associated with date.`,
+      typeName: 'aggressiveRemovalDAO',
```

Comment:
nit: aggressiveRemovalDataDAO, or remove the word data from the other DAO fields.